free-courses-vi.md

```diff
@@ -112,6 +112,7 @@
 ### Go
 
 * [Golang Cơ Bản](https://www.codehub.com.vn/Golang-Co-Ban)
+* [Golang cho người mới bắt đầu](https://www.youtube.com/playlist?list=PLVDJsRQrTUz5icsxSfKdymhghOtLNFn-k) - Code4Func
```

Comment:
We prefer to use the title as defined by the resource, please don't create your own title.

```suggestion
* [Lập trình Golang](https://www.youtube.com/playlist?list=PLVDJsRQrTUz5icsxSfKdymhghOtLNFn-k) - Code4Func
```